apps/aecore/lib/aecore/structures/account.ex

```diff
@@ -30,14 +33,22 @@ defmodule Aecore.Structures.Account do
 
   def empty, do: %Account{balance: 0, nonce: 0}
 
+  @spec new(account_payload()) :: Account.t()
+  def new(%{balance: balance, nonce: nonce}) do
+    %Account{
+      balance: balance,
+      nonce: nonce
+    }
+  end
+
   @doc """
   Builds a SpendTx where the miners public key is used as a sender (sender)
   """
   @spec spend(Wallet.pubkey(), non_neg_integer(), non_neg_integer()) :: {:ok, SignedTx.t()}
   def spend(receiver, amount, fee) do
     sender = Wallet.get_public_key()
     sender_priv_key = Wallet.get_private_key()
-    nonce = Map.get(Chain.chain_state().accounts, sender, %{nonce: 0}).nonce + 1
+    nonce = Account.nonce(Chain.chain_state().accounts, sender) + 1
```

Comment:
just nonce instead of Account.nonce or is it done for readability?

Reply:
Yes, is it done for readability.